allowances/migrations/1_deploy_allowance_modules.js

```diff
@@ -1,13 +1,13 @@
-const { deployTruffleContract } = require('@gnosis.pm/singleton-deployer-truffle');
-const AllowanceModule = artifacts.require("AllowanceModule");
+const { deployTruffleContract } = require("@gnosis.pm/singleton-deployer-truffle")
```

Comment:
my IDE was extremely unhappy about the JS formatting in this file so it formatted it